examples/aws_efa/README.md

```diff
@@ -0,0 +1,157 @@
+
+Elastic Fabric Adapter (EFA) is an AWS alternative to Nvidia infiniband that enables high levels of inter-node communications. It is specifically useful for distributed AI training and inference, which requires high network bandwidth across nodes.
+
+
+## TL;DR: enable EFA with SkyPilot
+
+You can enable EFA on AWS HyperPod/EKS clusters with an simple additional setting in your SkyPilot YAML:
+
+```yaml
+config:
+  kubernetes:
+    pod_config:
+      spec:
+        containers:
+        - resources:
+            limits:
+              vpc.amazonaws.com/efa: 4
+            requests:
+              vpc.amazonaws.com/efa: 4
+```
+
+
+
+## Enable EFA with HyperPod/EKS
+
+* On HyperPod (backed by EKS), EFA is enabled by default, and you don't need to do anything.
+* On EKS, you may need to enable EFA with the [official AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/node-efa.html).
+
+To check if EFA is enabled, run:
+```
+kubectl get nodes "-o=custom-columns=NAME:.metadata.name,INSTANCETYPE:.metadata.labels.node\.kubernetes\.io/instance-type,GPU:.status.allocatable.nvidia\.com/gpu,EFA:.status.allocatable.vpc\.amazonaws\.com/efa"
+```
+You can expect a sample output like:
+```
+NAME                           INSTANCETYPE      GPU   EFA
+hyperpod-i-0beea7c849d1dc614   ml.p4d.24xlarge   8     4
+hyperpod-i-0da69b9076c7ff6a4   ml.p4d.24xlarge   8     4
+...
+```
+
+## Access HyperPod and run distributed job with SkyPilot
+
+To accesss HyperPod and run distributed job with SkyPilot, see the SkyPilot [HyperPod example](../hyperpod-eks/).
+
+### Adding EFA configurations in SkyPilot YAML
+
+To enable EFA in SkyPilot YAML, you can specify the following section in the SkyPilot YAML:
+
+```yaml
+config:
+  kubernetes:
+    pod_config:
+      spec:
+        containers:
+        - resources:
+            limits:
+              vpc.amazonaws.com/efa: 4
+            requests:
+              vpc.amazonaws.com/efa: 4
+```
+
+This section is important for EFA integration:
+
+- `config.kubernetes.pod_config`: Provides Kubernetes-specific pod configuration
+- `spec.containers[0].resources`: Defines resource requirements
+  - `limits.vpc.amazonaws.com/efa: 4`: Limits the Pod to use 4 EFA devices
+  - `requests.vpc.amazonaws.com/efa: 4`: Requests 4 EFA devices for the Pod
+
+
+The `vpc.amazonaws.com/efa` resource type is exposed by the AWS EFA device plugin in Kubernetes. 
+To see how many EFA are available for each instance types that have EFA, see the [Network cards](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html#network-cards) list in the Amazon EC2 User Guide.
+
+Check the following table for the GPU and EFA count mapping for AWS `p4` and `p5` instance types:
+
+| Instance Type | GPU Type | #EFA |
+|---------------|----------|------|
+| p4d.24xlarge  | A100:8   | 4    |
+| p4de.24xlarge | A100:8   | 4    |
+| p5.48xlarge   | H100:8   | 32   |
+| p5e.48xlarge  | H200:8   | 32   |
+| p5en.48xlarge | H200:8   | 16   |
```

Comment:
Can't it be A10G? IIRC, the previous readme has A10G

Reply:
Added more GPU-EFA mappings.